packages/components/ng-ovh-sso-auth-modal-plugin/src/controller.js

```diff
@@ -1,4 +1,4 @@
-import $ from 'jquery';
+import 'whatwg-fetch';
```

Comment:
Is it required to work ? Or is it required because we want that the application that use it support old browser versions that don't provide `fetch` ?